tesseract_kinematics/core/include/tesseract_kinematics/core/kinematic_group.h

```diff
@@ -50,8 +50,8 @@ struct KinGroupIKInput
   EIGEN_MAKE_ALIGNED_OPERATOR_NEW
   // LCOV_EXCL_STOP
 
-  KinGroupIKInput(const Eigen::Isometry3d& p, std::string wf, std::string tl)
-    : pose(p), working_frame(std::move(wf)), tip_link_name(std::move(tl))
+  KinGroupIKInput(Eigen::Isometry3d p, std::string wf, std::string tl)
```

Comment:
Eigen types have to be passed by reference. If clang-tidy is complaining then add a nolint for the error it is giveing.

Reply:
If they every updated to 3.4 this will not be the case but currently is a limitation of  versions < 3.4.

Reply:
Reverted the change. I should've left that file alone 🙄